packages/core/src/components/message/test/warning/message.e2e.ts

```diff
@@ -0,0 +1,25 @@
+import { test } from 'stencil-playwright';
+import { expect } from '@playwright/test';
+
+test.describe('tds-message', () => {
+  test('Renders default Message correctly', async ({ page }) => {
```

Comment:
```suggestion
  test('renders warning message correctly', async ({ page }) => {
```
Suggest we use lower case for test names and update to correct "test" spec here and `warning` and then remove each image and update them so they have the correct name.